core/src/test/scala/org/apache/spark/memory/StaticMemoryManagerSuite.scala

```diff
@@ -43,12 +44,14 @@ class StaticMemoryManagerSuite extends MemoryManagerSuite {
 
   override protected def createMemoryManager(
       maxOnHeapExecutionMemory: Long,
-      maxOffHeapExecutionMemory: Long): StaticMemoryManager = {
+      maxOffHeapExecutionMemory: Long = 1000): StaticMemoryManager = {
+    val offHeapSize = conf.get(MEMORY_OFFHEAP_SIZE)
```

Comment:
Why read this conf? we always get default value here, don't we?

Reply:
Yeah, I made a mistake.